File: op/euler/js/problem1.js
function SumOfMultipleOf(num, div) {
    var sum = 0;
    if (num === 0 || div === 0) {
        return 0;
    }

    for (var i = 1; i < num; i++) {
        if (i % div == 0) {
            sum += i;
        }
    }

    return sum
}

function printSolution() {
    document.getElementById("answer").innerHTML = SumOfMultipleOf(1000, 3) + SumOfMultipleOf(1000, 5) - SumOfMultipleOf(1000, 15);
    return
}

File: op/euler-ruby/problem1.rb
sum = 0

def getMultiplierSum val, div
    sum = 0
    if (val < 1 || div < 1) then return sum end

    div.times do |i|
        if i%val == 0 then
            sum += i
        end
    end
    return sum
end

puts getMultiplierSum(3, 1000) + getMultiplierSum(5, 1000) -  getMultiplierSum(15, 1000)